### ISSUE TITLE
Suggestion: Tutorials?

### ISSUE THREAD

**Issue by XenoHoe** [petrolpark/Destroy#60]
I tried the mod and this is the first time I felt lost when trying to make something even with JEI.

While the mod sure has a very realistic simulation of chemistry, it can be quite overwhelming for a player new to the mod. The sea of reactions and recipes and chemicals seem quite non-naviagtable and to the point that I almost had no idea where to start. It feels like in its current state, the mod is only good for players who already have quite extensive knowledge in chemistry, and without the knowledge, it's like discovering chemistry from scratch all over again.

As a high-school student with a not-so extensive knowledge(especially in organic chemistry), the mod appears quite unplayable. While there is fun in re-inventing chemistry, I still would love at least some basic tutorial/guide on what to do, how to get started in the mod's game progression, how to practically utilize the recipes/reactions listed in JEI, how to troubleshoot the process if things doesn't work as intended or is too slow, etc.

While a new in-game tutorial system would be nice, it is understandable that it would take loads of work and isn't exactly for beta. So, maybe some written documents somewhere on the internet? It's OK if it's not top priority, but that would help out A TON.

**Comment by AngelOfCreation** [comment]
Agreed. I built a vat but I have no idea what to do with it. All the chemical reactions want chemicals, obviously, but I have no clue where to get them or how to put them in my vat.

**Comment by petrolpark** [comment]
An in-game tutorial system will come very far down the line.

**Comment by AngelOfCreation** [comment]
> An in-game tutorial system will come very far down the line.

Wow, what great planning on your part.
